apps/package.json

```diff
@@ -15,6 +15,7 @@
     "test:integration": "grunt integrationTest",
     "test:entry": "grunt karma:entry",
     "test:code-studio": "grunt codeStudioTest",
+    "flow": "flow src",
```

Comment:
Also, `flow` does not take a directory name, but a command. I think you want `flow check`. The directory that it will check is whichever one has a `.flowconfig` file.